src/components/AttachmentPicker/index.native.js

```diff
@@ -66,11 +65,7 @@ const documentPickerOptions = {
   * @return {Promise}
   */
 function getDataForUpload(fileData) {
-    let fileName = fileData.fileName || fileData.name || 'chat_attachment';
-    const fileExtension = `.${_.last(fileData.type.split('/'))}`;
-    if (!Str.endsWith(fileName, fileExtension)) {
-        fileName = `${fileName}${fileExtension}`;
-    }
+    const fileName = fileData.fileName || fileData.name || 'chat_attachment';
```

Comment:
I'm guessing that this `|| 'chat_attachment'` case never happens anymore. I think it is fine to leave it like this just in case and this will be changed anyway once we fix the issue https://github.com/Expensify/App/issues/11085